www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,112 @@
+---
+import { Markdown, Prism } from 'astro/components';
+import BlogHead from '../../components/BlogHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+import BlockQuote from '../../components/BlockQuote.astro';
+import GoogleAnalytics from '../../components/GoogleAnalytics.astro';
+import Note from '../../components/Note.astro';
+import Shell from '../../components/Shell.astro';
+
+let title = 'Astro 0.18 Release';
+let description = `Astro 0.18 is our biggest release since Astro launch. It includes a new responsive hydrator, named slots, global stylesheets, and two new renderers. Ready on to learn more on this release.`;
+let publishDate = 'Tuesday, July 27 2021';
+let author = 'matthew';
+let heroImage = '/social.jpg';
+let permalink = 'https://astro.build/blog/astro-0-18';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BlogHead title={title} description={description} permalink={permalink} />
+    <style>
+      img {
+        max-width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author={author} heroImage={heroImage} publishDate={publishDate}>
+        <Markdown>
+          After the success of Astro's launch we spent some time fixing bugs and taking user feedback on what to tackle next. Now some of the features that people requested are starting to get released.
+          
+          Astro 0.18 is the culmination and is the biggest release to date. This post highlights only some of what it brings.
+
+          ![Code example that shows off using the new client:media hydrator.](/assets/blog/astro-018/responsive-hydration.png)
```

Comment:
I'd recommend removing the `---` code fence, and adding an HTML 1-line comment explaining what's happening for someone quickly browsing. Something like:

```
<!-- Hydrate the sidebar on mobile devices, only! -->
<Sidebar client:media="(max-width: 700px)" />
```